Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -6,6 +6,40 @@
 //  Copyright © 2016 SwifterSwift
 //
 
+#if canImport(CoreLocation)
```

Comment:
I think those extensions should be under the CoreLocation folder. Eventhough they are arrays they involve CoreLocation :)

Reply:
ah ok. I couldn't decide if it was array or corelocation.

Reply:
Is still an array extension, but it should be under the CoreLocation extensions module :)

Reply:
@LucianoPAlmeida I fixed all your suggestions. The only thing I wasn't sure of is if I put it in corelocationextensions or it's seperate file. I chose the later but let me know. thanks!

Reply:
I think what you did is fine :)) 
Can you just check why the CI is failing?